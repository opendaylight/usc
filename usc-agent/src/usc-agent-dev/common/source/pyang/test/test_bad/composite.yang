module composite {
  namespace "urn:test:composite";
  prefix c;

  include submodule1;
  include submodule2;
  include submodule4;
  include submodule5;

  include crypto-base; // error: crypto-base is a module

  typedef type1 { // error: defined in submodule1
    type int8;
  }

  identity id1; // error: defined in submodule1

  feature f1; // error: defined in submodule1

}
