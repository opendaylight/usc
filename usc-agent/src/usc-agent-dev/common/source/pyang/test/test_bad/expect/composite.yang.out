composite.yang:10: error: BAD_INCLUDE
composite.yang:12: error: TYPE_ALREADY_DEFINED
composite.yang:16: error: IDENTITY_ALREADY_DEFINED
composite.yang:18: error: FEATURE_ALREADY_DEFINED
submodule1.yang:11: error: TYPE_ALREADY_DEFINED
submodule1.yang:19: error: GROUPING_ALREADY_DEFINED
submodule2.yang:7: error: BAD_IMPORT
submodule2.yang:7: warning: UNUSED_IMPORT
submodule2.yang:11: error: MISSING_INCLUDE
submodule5.yang:9: error: NODE_NOT_FOUND
submodule5.yang:10: error: GROUPING_NOT_FOUND
